### PR TITLE
Arch doc with images

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

File: docs/README-architecture.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+The Undercloud implemented via OpenShift, and OpenStack is deployed via a pod.
+![Architecture](./arch.png)
+
+
+Custom Resource Definitions (CRDs)
+----------------------------------
+The OSP Director Operator creates a set of Custom Resource Defitions on top of OpenShift to manage resources normally created by the TripleO's Undercloud. These CRDs are split into two types for hardware provisioning and software configuration which aligns well with TripleO's Undercloud Lite architecture.
+![CRDs](./crds.png)
+
+The left side of this diagram describes our hardware provisioning CRDs including OvercloudNet, ControlPlane, and BaremetalSet. These CRDs will allow administrators to provision and scale out virtual and baremetal hardware for an OSP deployment. Virtual machines for the OSP control plane are provisioned via Kubevirt(OpenShift Virtualization). Baremetal for OSP compute hosts is provisioned via Metal3. And OvercloudNet provides a simple IPAM solution for IPv4 and IPv6 static IP assignments.
```
Comment (abays) [inline docs/README-architecture.md:17]: I think we should rename all instances of `OvercloudNet` to `OpenstackNet` to be more consistent with the diagrams

Comment (dprince) [reply docs/README-architecture.md]: fixed in latest commit, thanks

File: docs/README-architecture.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+OSP Director Operator Architecture
+==================================
+
+Architecture:
+-------------
+This shows the OSP architecure that we currently support from a high level. The Controllers are hosted as VM on OpenShift Virtualization/Kubevirt. Baremetal Hosts are used for OSP Computes.
```
Comment (stuggi) [inline docs/README-architecture.md:6]: "The Controllers are hosted as VM" -> VMs? or virtual machines plural?

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**